docs/tls.md

```diff
@@ -30,7 +30,10 @@ For instance, to use self-signed certificates created with [mkcert](https://gith
     ```diff
     php:
       environment:
-    +    CADDY_SERVER_EXTRA_DIRECTIVES: "tls /etc/caddy/certs/tls.pem /etc/caddy/certs/tls.key"
+    +    CADDY_EXTRA_CONFIG: |
+    +      https:// {
+    +          tls /etc/caddy/certs/tls.pem /etc/caddy/certs/tls.key
+    +      }
```

Comment:
There is incorrect formatting here (Caddyfile uses tabs for indentation)

```suggestion
    +      https:// {
    +      	tls /etc/caddy/certs/tls.pem /etc/caddy/certs/tls.key
    +      }
```

IMO it looks wrong visually (as if a single space is used for indentation, not a tab).

Reply:
In suggestion it looks better than it renders in Markdown
![image](https://github.com/user-attachments/assets/d1c689c4-0b12-4708-8979-de108983e9fd)